connection-service/Dockerfile

```diff
@@ -1,15 +1,12 @@
-FROM docker.io/dunedaq/c8-minimal:latest
+FROM docker.io/almalinux:9
 
 RUN yum clean all \
- && yum -y install python3-pip \
- && yum -y install python3-devel \
+ && yum -y install python3-pip python3-devel git \
  && yum clean all
 
-COPY connection-flask.py /
-COPY requirements.txt /
 
-RUN pip3 install --upgrade pip
-RUN pip3 install -r /requirements.txt
+RUN git clone --depth 1 https://github.com/DUNE-DAQ/connectivityserver.git
```

Comment:
I think this is fine, however for the future, could this be made such that it uses the correct version of the connectivity service GitHub? We probably don't want to clone `develop`'s head every time we rebuild the image. So for good measure, I'd suggest tagging the connectivity service repo (format is `vX.X.X`) and adding it here with a: `-b the-version`.

Reply:
Good point, I've updated it to specify v1.0.0